src/comm/Connection.h

```diff
@@ -52,6 +52,7 @@ namespace Comm
 #define COMM_REUSEPORT          0x40 //< needs SO_REUSEPORT
 /// not registered with Comm and not owned by any connection-closing code
 #define COMM_ORPHANED           0x40
+#define COMM_BIND_NOW           0x80 // Skips IP_BIND_ADDRESS_NO_PORT sockopt to ensure local port is assigned
```

Comment:
@rousskov BTW, had noticed multiple comm flags using 0x40 in Connection.h. If you think it makes things simpler for you, I can push another change to have the new flag COMM_BIND_NOW introduced here to use the next bit 0x100.

Reply:
Your call. If I do not see that commit, I will do it myself. It was one of the changes on my to-do list. I still hope to work on this later today.

Reply:
Just pushed a commit to change it to 0x100.